src/layout/TopBar/TopBar.js

```diff
@@ -55,12 +91,33 @@ function TopBar({
         </Link>
 
         <div className={classes.menuRight}>
-          <Link to={routes.USER_MANAGEMENT}>
-            <IconButton aria-label="user-management" color="inherit">
-              <GroupIcon fontSize="large" className={classes.menuIcon} />
-            </IconButton>
-          </Link>
-          <IconButton aria-label="logout" color="inherit" onClick={handleLogoutClick}>
+          {isSuperAdmin && (
+          <TextField
+            className={classes.globalFilter}
+            variant="outlined"
+            fullWidth
+            id="org"
+            label="Organization"
+            select
+          >
+            <MenuItem>
```

Comment:
Why is only None option hard coded here for Super Admin?

Reply:
Because we don't have any api to fetch all org names

Reply:
There should be an API that we already use in TP as well as Insights code. So it should be similar here as well.

Reply:
I tried and it was throwing 404 confirmed with BE as well that there's no API in buildly.io to fetch all orgs

Reply:
If we want a global admin filter we need to have a list of organizations. Without it, you cannot create a dropdown for global admin.